tools/crm_mon.c

```diff
@@ -1458,6 +1458,8 @@ main(int argc, char **argv)
                 g_set_error(&error, PCMK__EXITC_ERROR, CRM_EX_USAGE,
                             "--daemonize requires at least one of --output-to and --external-agent");
                 return clean_up(CRM_EX_USAGE);
+            } else if (args->output_ty == NULL) {
+                args->output_ty = strdup("html");
```

Comment:
I have mixed feelings about this since we already have a rather contorted default, but I'm OK with it if you update the help for --daemonize

Reply:
I'd be happy to drop this patch instead.